src/compiler/diagnosticMessages.json

```diff
@@ -671,6 +671,10 @@
         "category": "Error",
         "code": 1215
     },
+    "Identifier expected. '__esModule' is reserved as a marker for transpiled ES2015 module.": {

```

Comment:
I think "Identifier expected" is a confusing bit of information that probably shouldn't be reported here.